.github/workflows/ci.yml

```diff
@@ -39,3 +39,59 @@ jobs:
         run: ./lint --disable=W
       - name: Check docstrings
         run: pydocstyle --match-dir='(?!v2|cuKING).*' gnomad_qc
+  docs:
+    name: Build documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
what does this line do?

Reply:
allows us to import from `gnomad_qc.sample_qc.cuking`